selfdrive/test/test_onroad.py

```diff
@@ -68,7 +68,7 @@
     "system.sensord.pigeond": 6.0,
   },
   "tizi": {
-     "./boardd": 19.0,
```

Comment:
why would this reduce the cpu here? `Panda::list` is only called while offroad and this tests onroad cpu

Reply:
reverted